crates/polars-core/src/series/implementations/boolean.rs

```diff
@@ -25,10 +25,12 @@ impl private::PrivateSeries for SeriesWrap<BooleanChunked> {
     fn _dtype(&self) -> &DataType {
         self.0.ref_field().data_type()
     }
-    fn _clear_settings(&mut self) {
-        self.0.clear_settings()
+    fn _get_flags(&self) -> u8 {
```

Comment:
Nice! On a next PR we can remove `_set_sorted_flag` and `_get_sorted_flag` from the `dyn Trait` and simply dispatch that from the `Series` struct to these `_get_flags` and `_set_flags`. Saves 2 entries in the vtable.